src/js/letters/containers/AddressSection.jsx

```diff
@@ -53,7 +81,7 @@ export class AddressSection extends React.Component {
             Downloaded documents will list your address as:
           </p>
           <div className="address-block">
-            <h5 className="letters-address">{(destination.fullName || '').toLowerCase()}</h5>
+            <h5 className="letters-address">{(this.props.fullName || '').toLowerCase()}</h5>
```

Comment:
So this took me a long time to figure out what `fullName` belonged to. Can we change this to `recipientName` or something to make it clear that it sort of "belongs" in the address? Or does that not make as much sense?

Reply:
Sure--it comes to use in the response as `fullName` but sometimes these weirdly-named response properties are worth renaming in our props.

Reply:
Yep, can change.